File: js_project/Arrays/EvenOddSeperation.js
let a= new Array(10).fill(0);
let b= new Array(10).fill(0);
for (let i=0; i<a.length; i++){
    a[i]= i+1;
    if(i%2==0){
        console.log(i);
    }
}
console.log("even sequence");
for (let i=0; i<b.length; i++){
    b[i]=i+1;
    if(i%2!=0){
        console.log(i);
    }
}
console.log("odd sequence")

File: java_project/Array/EvenOddSeperation.java
package Array;

public class EvenOddSeperation {
    public static void main(String[] args) {
        int a[]=new int[10];
        int b[]=new int[a.length];
        for (int i=0; i<a.length;i++){
            a[i]=i+1;
            //System.out.print(a[i]+"\t");
            if (i%2==0){
                System.out.print(i+"\t");
            }
        }
        System.out.println("even sequence");
        for (int i=0; i<b.length;i++){
            b[i]=i+1;
            //System.out.print(b[i]+"\t");
            if (i%2!=0){
                System.out.print(i+"\t");
            }
        }
        System.out.println("Odd sequence");

    }
}
